kazoo/recipe/lock.py

```diff
@@ -54,7 +54,13 @@ def __init__(self, client, path, identifier=None):
         :param identifier: Name to use for this lock contender. This
                            can be useful for querying to see who the
                            current lock contenders are.
-
+        :param node_name: Node name, after the contender UUID, before the
+                          sequence number. Involved in read/write locks. For a
+                          normal (exclusive) lock, leave unset.
+        :param exclude_names: Node names which exclude this contender when
+                              present at a lower sequence number. Involved in
+                              read/write locks. For a normal (exclusive) lock,
+                              leave unset.
```

Comment:
I'd consider moving the comment that these are for read/write locks to the front, so that uninterested people know they can ignore them.